package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@s9a/tape",
+  "version": "0.0.0",
+  "lockfileVersion": 1
+}
```

Comment:
`npm init` as `0.0.0`  so that first bump via `npm version minor` will be isolated bump to `0.1.0` for first release on npm